### PR TITLE
test(sidenav): add e2e tests

### PR DESCRIPTION
**Pull request by robindijkhof**

### CODE REVIEW

File: e2e/components/sidenav-e2e.spec.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import {browser, by, element, protractor} from 'protractor';
```
Comment (mmalerba) [inline e2e/components/sidenav-e2e.spec.ts]: import {browser, by, element, ExpectedConditions} from 'protractor';
const EC = ExpectedConditions;

File: e2e/components/sidenav-e2e.spec.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+const EC = protractor.ExpectedConditions;
+
+
+describe('sidenav', () => {
```
Comment (mmalerba) [inline e2e/components/sidenav-e2e.spec.ts]: nit: to match style used elsewhere in the project, indent size should be 2 spaces

File: src/e2e-app/sidenav/sidenav-e2e.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<section>
```
Comment (mmalerba) [inline src/e2e-app/sidenav/sidenav-e2e.html]: nit: indent = 2 spaces

File: src/e2e-app/sidenav/sidenav-e2e.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import {Component} from '@angular/core';
+
+
+@Component({
+    moduleId: module.id,
```
Comment (mmalerba) [inline src/e2e-app/sidenav/sidenav-e2e.ts]: nit: indent = 2 spaces

**Comment by robindijkhof** [comment]
@mmalerba I do not have much experience on contributing to open source projects. The Travis build fails on something completely out of my scope. Any suggestions?

**Comment by mmalerba** [comment]
I think that's just test flakiness. We randomized the execution order of the tests and it uncovered some issues that we haven't completely fixed yet

**Pull request merged**